src/client/actions/GuildEmojiUpdate.js

```diff
@@ -0,0 +1,27 @@
+const Action = require('./Action');
+const Constants = require('../../util/Constants');
+
+class EmojiUpdateAction extends Action {
+  handle(data, guild) {
+    const client = this.client;
+    data.emojis.forEach(emoji => {
```

Comment:
instead of forEach, could you just use `for (.. of ..)` instead? It's more consistent with the rest of d.js (please apply changes throughout the PR)